GET_STARTED.md

```diff
@@ -62,6 +65,7 @@ spec:
   container:
     name: storage-initializer
     image: quay.io/$USER/oci-storage-initializer:${VERSION}
+    imagePullPolicy: IfNotPresent # NOT FOR PROD but allow easier testing of local images with KinD (just remove for prod)
```

Comment:
I remember I did the same for the model registry csi but this time I forgot. Thanks a lot!